src/linkify-react.js

```diff
@@ -98,7 +98,7 @@ function linkifyReactElement(element, opts, elementId = 0) {
 	return React.cloneElement(element, newProps, children);
 }
 
-var Linkify = React.createClass({
+class Linkify extends React.Component {
```

Comment:
Replacing `createClass` with the class style definition wasn't in the scope of this change, but the tests wouldn't pass no matter what else I tried.

Reply:
❤️ ES6 syntax

Reply:
but dont we need a constructor function now?

Reply:
Nope. `render` is the only function you need when declaring a react component

Reply:
I know render is the only one with the ‘createClass’ syntax but I think using class + extends  the constructor is needed to call ‘super’ maybe I’m wrong just wanted to double check

Reply:
There's no `state` being initalized, so I don't see why we would need to call `super` in the constructor.

Reply:
https://reactjs.org/docs/react-component.html#constructor

> If you don’t initialize state and you don’t bind methods, you don’t need to implement a constructor for your React component.

Okay, now I'm sure that we don't need the constructor 💯

Reply:
🎉

Reply:
if all you have is render, why not go real short hand? 

```js
const Linkify = props => {
  return ()
}
```

but on that note, why make this change? I would strive for consistency with other components, so if everything uses `React.createClass` then I think it should stay that way...

Reply:
agree... test fail even in master... no CI here so maybe it's safe to ignore it? 🤷‍♂️

Reply:
Hm, okay :-? Consistency it is. I undo the change and leave it like that :)